Lib/defconQt/fontView.py

```diff
@@ -1243,7 +1259,7 @@ def _selectionChanged(self, selection):
                 count = selection
                 text = ""
             if not count == 0:
-                text = "%s(%d selected)" % (text, count)
+                text = self.tr("{0}({1} selected)").format(text, count)
```

Comment:
Hm, it won't fail, but I also see nothing in Linguist indicating that I can pluralize...